libs/java/service/src/main/java/ca/bc/gov/tno/services/converters/ZonedDateTimeDeserializer.java

```diff
@@ -0,0 +1,49 @@
+package ca.bc.gov.tno.services.converters;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+
+/**
+ * Custom {@link ZonedDateTime} deserializer.
+ */
+public class ZonedDateTimeDeserializer extends JsonDeserializer<ZonedDateTime> {
+
+  /**
+   * @param jsonParser for extracting the date in {@link String}
+   *                   format.
+   * @param context    for the process of deserialization a single
+   *                   root-level value.
+   * @return {@link ZonedDateTime} object of the date.
+   * @throws IOException throws I/O exceptions.
+   */
+  @Override
+  public ZonedDateTime deserialize(JsonParser jsonParser, DeserializationContext context)
```

Comment:
Had to add these to handle the various errors occurring during deserialization.  Java can't handle dates... it's embarrassing!